README.md

```diff
@@ -0,0 +1,109 @@
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+
+- [💻 Getting Started](#getting-started)
+
+  - [Usage](#usage)
+
+- [👥 Authors](#authors)
+
+- [⭐️ Show your support](#support)
+
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  Portfolio project
+  <a name="about-project"></a>
+The following project consist of demostrating the usage of github best practices, Linsts configs and how to write source code documentation in professional manner.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Box icons</a></li>
+    <li><a href="https://reactjs.org/">Google fonts</a></li>
+  </ul>
+</details>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Usage
+To run the project click on the HTML document on your local machine or use the live server VS.code extension
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- solid Linting  
+- responsive design 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+The live demo link is currently unavailable due to technical stuff and maintenance, please be patient as we are working on something presentable 
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@DjibrilMugisho](https://github.com/DjibrilM?tab=overview&from=2023-04-01&to=2023-04-26)
+- Twitter: [@djibrilM](https://twitter.com/DjibrilMugisho)
+- LinkedIn: [DjibrilMugisho](https://www.linkedin.com/in/djibril-mugisho-509042241/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project don't for get to leave a star, and leave your feedback.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for all the inspiration and great support offered for this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I noticed that one of the links you included, specifically the M.I.T link, is broken and no longer functional. :wink: 
As you know, links are an essential part of any project, and it's crucial to ensure that they are functional and lead to the intended content. In this case, the broken link may hinder the ability of others to access important information related to your project. Kindly fix this :point_down: 
![Screenshot from 2023-04-28 15-44-09](https://user-images.githubusercontent.com/88894525/235171019-45c39e1c-8ca3-4482-93ad-fac6c7079ad9.png)